src/redux/utilities.js

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const appID = 'GahFETqP7Dmi6zLej4E7';
+
+export const addBook = createAsyncThunk('Books/addBook', async (book) => {
+  try {
+    await axios.post(
+      `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books`,
+      book,
+    );
+  } catch (e) {
+    throw new Error(e);
+  }
+  return book;
+});
```

Comment:
- [ ] Great job so far @katarighe fixing the first error the previous reviewer noticed. However, I noticed that currently when you add a book for the first time it works without errors. But when you add a second or third book, it throws an error. :+1: 
- [ ] I have attached a video below for your reference so kindly fix this so that a user can add books and be able to retrieve all of them back from the API without any errors :wink: 
- Video Reference

[Screencast from 08-23-2023 04:13:50 PM.webm](https://github.com/katarighe/bookstore/assets/102699958/37ec4d23-68b5-4d27-ab3b-88ec6f52e03b)

Reply:
@NanahAJ It works for me without errors. I can add 3 or 4 books.